doc/vim-go.txt

```diff
@@ -1759,6 +1764,37 @@ After opening vim, run `:echo $PATH`, the output must be your current `$PATH`
 plus `$GOPATH/bin` (the location where |:GoInstallBinaries| installed the
 binaries).
 
+                                                               *go-guru-scope*
+What is the guru scope and how do I set it?~
```

Comment:
What follows this question is salient, but this leading question doesn't add value for the reader. Let's elide it.

Reply:
How would you phrase it?

Reply:
Just remove it; it's not necessary.

Reply:
Right now all the FAQ items have a header; without it, it would look a bit weird I think?

Reply:
Ah, you're right. I didn't realized that.